src/modules/services_manager_service.rb

```diff
@@ -47,16 +45,14 @@ def read
         load_units
 
         @supported_unit_files = unit_files.select do |_, status|
-          Status::SUPPORTED_STATES.member?(status)
+          status != Status::MASKED # masked services should not been shown at all
```

Comment:
I guess you're doing this because we handle all the status properly now, right? I such a case, is still the `SUPPORTED_STATES` constant being used?